test/fuzz/fuzz-catalog/systemd.ru.catalog

```diff
@@ -0,0 +1,403 @@
+# SPDX-License-Identifier: LGPL-2.1+
+#
+#  Copyright © 2013-2017 Sergey Ptashnick
```

Comment:
I thought that all "copyright" lines were removed in https://github.com/systemd/systemd/issues/9320, but, apparently some of them are still here. @nnz1024 would it be OK if removed this from `systemd.ru.catalog`?

Reply:
In the end I decided to use `systemd.pl.catalog` here. Anyway, @nnz1024 it would be great if you could drop the copyright line there.